src/opendp/test/multi_height05.defok

```diff
@@ -0,0 +1,20 @@
+VERSION 5.8 ;
+NAMESCASESENSITIVE ON ;
```

Comment:
If the DEF parser can't read it without that statement - indicating another bug - then the issue is moot for now and will have to be revisited after the DEF parser is fixed.

That being said, this is a test input file. It has nothing to do with what OpenDB outputs. DEF has a defined standard. Fixing the output from OpenDB and fixing this specific test input are orthogonal issues. Fixing the rest of the instances of this issue in other tests is an orthogonal issue - one has to fix bugs as they find them rather than ignore them just because they exist elsewhere.

Reply:
Ignore previous statement, this is a defok file not a def file.